components/apify/actions/get-dataset-items/get-dataset-items.mjs

```diff
@@ -5,7 +5,7 @@ export default {
   key: "apify-get-dataset-items",
   name: "Get Dataset Items",
   description: "Returns data stored in a dataset. [See the documentation](https://docs.apify.com/api/v2/dataset-items-get)",
-  version: "0.0.2",
+  version: "0.0.3",
```

Comment:
Do we really need to change version of all actions, when we are updating just scrape singe URL?